resources/js/components/GlobalSearch.vue

```diff
@@ -0,0 +1,282 @@
+            if (!data.length) this.topicsFlag = false;
+
+        },
+        async getPeople () {
+            if (!this.peopleFlag) return;
+            this.paginationPeople += 1;
+            const { members, meta } = await people.getPeopleLike(this.inputValue, this.paginationPeople);
+            this.peopleCount = meta.total;
+            this.peopleList = [...this.peopleList, ...members];
+            this.loading = false;
+            if (!members.length) this.peopleFlag = false;
+        },
+        async getLocations () {
+            if (!this.locationFlag) return;
+            this.paginationLocation += 1;
+            const { locations, meta } = await location.getLocationLike(this.inputValue, this.paginationLocation);
+            this.locationCount = meta.total;
+            this.locationList = [...this.locationList, ...locations];
+            this.loading = false;
+            if (!locations.length) this.locationFlag = false;
+        },
+        lazyHandle () {
+            this.debounce && clearTimeout(this.debounce);
+            this.debounce = setTimeout(
+                () => {
+                    if (this.activeTab === 'Themen') this.getTopics();
+                    if (this.activeTab === 'Orte') this.getLocations();
+                    if (this.activeTab === 'Personen') this.getPeople();
+                }, 500);
+        },
+        changeTab (item) {
+            this.activeTab = item;
+            setTimeout(() => this.createObserver(), 0);
+        },
+        inputHandle () {
+            this.topicsFlag = true;
+            this.locationFlag = true;
+            this.peopleFlag = true;
+            this.topicsList = [];
+            this.locationList = [];
+            this.peopleList = [];
+            this.debounce && clearTimeout(this.debounce);
+            if (this.inputValue) {
+                this.loading = true;
+                this.debounce = setTimeout(
+                    () => {
+                        this.getTopics();
+                        this.getLocations();
+                        this.getPeople();
+                    }, 1000);
+            }
+        }
+    }
+};
+</script>
+
+<template>
+    <div class="ris-global-search">
+        <div
+            class="ris-search ris-global-search-input"
+            :class="{ 'focused': onFocus }">
+            <button @click="onFocus ? focusHandle(false) : focusHandle(true)" class="ris-search__button">
+                <span v-if="onFocus" class="ris-i ris-i_back" />
+                <span v-else class="ris-i ris-i_search"/>
+            </button>
+            <input
+                type="search"
+                class="ris-search__input ris-global-search-input__item"
+                :class="{ 'focused': onFocus }"
+                v-model="inputValue"
+                ref="globalSearchInput"
+                @focus="focusHandle(true)"
+                @input="inputHandle"
+                :placeholder="inputPlaceholder">
```

Comment:
why do you need a separate property for placeholder? i don't see that it gets changed somewhere

Reply:
is removed